dotnet/packages/Microsoft.TeamsAI/Microsoft.TeamsAI.Tests/AITests/AssistantMessageTest.cs

```diff
@@ -31,8 +31,8 @@ public void Test_Constructor()
             // Assert
             Assert.Equal(content, assistantMessage.MessageContent);
             Assert.Equal("message", assistantMessage.Content);
-            Assert.Equal(1, assistantMessage.AttachedFiles!.Count);
-            Assert.Equal("fileId", assistantMessage.AttachedFiles[0].FileInfo.Id);
+            Assert.Single(assistantMessage.AttachedFiles!);
```

Comment:
Should this file be renamed to AssistantMessageTests.cs?

Reply:
renamed